### DISCUSSION TITLE
Use images of different architecture

### DISCUSSION THREAD

**Discussion by sj14** [siderolabs/talos#6972] [Q&A] [answered]
Hello everyone,

I switched some while ago from k3os to talos. Back there on k3os I could run amd64 images on an arm64 k3os installation.
I think the magic was provided by `binfmt_misc`. Are there any plans to support the same with talos?

Thanks for the project and best regards,
Simon

**Comment by smira** [comment, marked as answer]
Talos tries to follow [KSPP](https://kernsec.org/wiki/index.php/Kernel_Self_Protection_Project/Recommended_Settings), and `binfmt_misc` is not considered secure enough, so it's not enabled in Talos.
